src/index.ts

```diff
@@ -17,10 +17,11 @@
 // ** All changes to this file may be overwritten. **
 
 import * as v1beta1 from './v1beta1';
+import * as v1 from './v1';
 const WebRiskServiceV1Beta1Client = v1beta1.WebRiskServiceV1Beta1Client;
-export {v1beta1, WebRiskServiceV1Beta1Client};
+export {v1, v1beta1, WebRiskServiceV1Beta1Client};
 // For compatibility with JavaScript libraries we need to provide this default export:
 // tslint:disable-next-line no-default-export
-export default {v1beta1, WebRiskServiceV1Beta1Client};
+export default {v1, v1beta1, WebRiskServiceV1Beta1Client};
```

Comment:
I don't understand what this line of code does.  Do you have a good grip on why/if this is needed?

Reply:
This approach is necessary to ensure that the module has the same export signature as the JavaScript library did prior to the TypeScript conversion:

```bash
{
  TextToSpeechClient: [Function: TextToSpeechClient],
  v1beta1: { TextToSpeechClient: [Function: TextToSpeechClient] },
  v1: [Circular],
  default: {
    TextToSpeechClient: [Function: TextToSpeechClient],
    v1beta1: { TextToSpeechClient: [Function: TextToSpeechClient] },
    v1: [Circular]
  }
}
```